winapps/__init__.py

```diff
@@ -112,6 +112,20 @@ def _none_on_value_not_set(value: Any) -> Any:
 })
 
 
+def dateCheck(d):
```

Comment:
I use PEP8 style, so, please, rename it to `date_check`, or, probably, to just `to_date`.